### PR TITLE
Add docs related to am's container

### PR DESCRIPTION
**Pull request by hatchan**
# Checklist

- [x] Changelog updated

### CODE REVIEW

**Review by mellowagain — APPROVED** [review]

**Review by IvanMerrill — APPROVED** [review]

File: Dockerfile.proxy [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+RUN apt-get update \
+    && apt-get install -y --force-yes --no-install-recommends ca-certificates \
+    && apt-get clean \
+    && apt-get autoremove \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY ${TARGETOS}/${TARGETARCH}/am /app/am
+
+ENV LISTEN_ADDRESS="0.0.0.0.6789"
```
Comment (brettimus) [inline Dockerfile.proxy:23]: this should fallback to `:6789` right?

Comment (brettimus) [reply Dockerfile.proxy:23]: (i see a period)

**Review by brettimus — APPROVED** [review]
thank you for making the proxy dockerfile <3

File: Dockerfile.proxy (hunk 1 above)
Comment (hatchan) [reply Dockerfile.proxy:23]: Good catch :+1:

**Pull request merged**